tutorial/template-tutorial.md

```diff
@@ -1,45 +1,87 @@
 
-# Part 2 - <_Part Title_>
+Before you start the tutorial, complete the following steps:
 
-{
-Continue the design approach you chose in the previous part and continue it through to the end of the tutorial.
-}
+* Prerequisite 1
+* Prerequisite 2
+* Prerequisite 3
 
-# What you've learned
+## Part 1 - {Description}
 
-{Summarise what knowledge the reader has gained by completing the tutorial, including a summary of each part's Goals (this is a good way to validate whether your tutorial has covered all you need it to.)}
+Feel free to use this section to summarize what users will complete in the following steps.
+
+### Step 1 - {Task name}
+
+{Provide background information for the task here. You might want to define any terminology here and provide any overview of this task's purpose.
+Include any code samples, snippets, or screenshots that will help your user complete the task}.
+
+{If users should see a result after complete this step, make sure to include it.}
+
+#### Substep 1 - {Subtask name}
+
+{Beware of asking your users to do too much in one step. If needed, break a step into one or two smaller substeps.}
+
+### Step 2 - {Task name}
+
+{Provide background information for the task here. You might want to define any terminology here and provide any overview of this task's purpose.
+Include any code samples, snippets, or screenshots that will help your user complete the task}.
+
+{If users should see a result after complete this step, make sure to include it.}
+
+#### Substep 1 - {Subtask name}
+
+{Beware of asking your users to do too much in one step. If needed, break a step into one or two smaller substeps.}
+
+## Part 2 - {Description}
+
+Feel free to use this section to summarize what users will complete in the following steps.
+
+### Step 1 - {Task name}
+
+{Provide background information for the task here. You might want to define any terminology here and provide any overview of this task's purpose.
+Include any code samples, snippets, or screenshots that will help your user complete the task}.
+
+{If users should see a result after complete this step, make sure to include it.}
+
+#### Substep 1 - {Subtask name}
+
+{Beware of asking your users to do too much in one step. If needed, break a step into one or two smaller substeps.}
+
+### Step 2 - {Task name}
+
+{Provide background information for the task here. You might want to define any terminology here and provide any overview of this task's purpose.
+Include any code samples, snippets, or screenshots that will help your user complete the task}.
+
+{If users should see a result after complete this step, make sure to include it.}
+
+#### Substep 1 - {Subtask name}
+
+{Beware of asking your users to do too much in one step. If needed, break a step into one or two smaller substeps.}
+
+## Summary
+
+{Use this section to summarize tasks the users completed in the tutorial and to provide further reading and resources.}
```

Comment:
**Response to Cameron's comment (non-blocking):** I think it can go in either section. It depends on the context. For example, if I want to point out an interesting take-away or mention something else they could do, I could possibly include the link there if it makes sense.